hbase-common/src/test/java/org/apache/hadoop/hbase/util/TestAvlUtil.java

```diff
@@ -54,9 +55,8 @@ public void testAvlTreeCrud() {
     final TreeMap<Integer, Object> treeMap = new TreeMap<>();
     TestAvlNode root = null;
 
-    final Random rand = new Random();
     for (int i = 0; i < NELEM; ++i) {
-      int key = rand.nextInt(MAX_KEY);
+      int key = RNG.nextInt(MAX_KEY);
```

Comment:
Ditto, since we do not have a fixed seed here, let's just use ThreadLocalRandom.